requirements/requirements_docs.txt

```
@@ -22,4 +22,4 @@ sphinx-gallery==0.11.1
 sphinx-notfound-page==0.8.3
 sphinxcontrib-websupport==1.2.4
 sphinxemoji==0.2.0
-vtk==9.0.3
+vtk==9.2.2
```

```suggestion
vtk==9.0.3
```
Watch out not to commit these changes =)